linux_os/guide/system/accounts/accounts-restrictions/account_expiration/account_emergency_expire_date/rule.yml

```diff
@@ -43,6 +43,7 @@ references:
     nist-csf: DE.CM-1,DE.CM-3,PR.AC-1,PR.AC-4,PR.AC-6
     srg: SRG-OS-000123-GPOS-00064,SRG-OS-000002-GPOS-00002
     stigid@ol8: OL08-00-020270
+    stigid@ol7: OL07-00-010271
```

Comment:
Please ensure that these keys are sorted.

Reply:
What a dumb mistake sorry. Addressed 👍🏻